packages/babel-plugin/src/utils/css-builders.tsx

```diff
@@ -184,5 +184,37 @@ export const buildCss = (node: t.Expression, meta: Metadata): CSSOutput => {
     return extractObjectExpression(node, meta);
   }
 
+  if (t.isIdentifier(node)) {
+    const binding = meta.parentPath.scope.getBinding(node.name);
+    if (
+      !binding ||
+      !t.isVariableDeclarator(binding.path.node) ||
+      !t.isExpression(binding.path.node.init)
+    ) {
+      throw new Error();
```

Comment:
Should we throw some meaning full error here with some description? 🤔

Reply:
we probably should - if it's ok with you i'll make another issue to come back and improve all these errors in one pass?

Reply:
yeah no worries man. we can do it later 😄